app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Visitor's vaccine has expired.")
+        if not visitor.get("wearing_a_mask", True):
+            raise NotWearingMaskError("Visitor is not wearing a mask.")
+        return f"Welcome to {self.name}"
```

Comment:
The `visit_cafe` method should return a string when the visitor is allowed to enter the cafe. Currently, it returns nothing. Update the method to return `f"Welcome to {self.name}"` when all conditions are met.